CHANGES.rst

```diff
@@ -1,12 +1,16 @@
 Changes
 *******
 
-TBD (unreleased)
+0.11.0 (2023-06-13)
 ================
```

Comment:
```suggestion
===================
```

Reply:
For reference, the underline needs to match the length of the line; Sphinx will complain otherwise.

Reply:
Ok changed